src/Symfony/Component/Validator/Resources/translations/validators.en.xlf

```diff
@@ -426,6 +426,10 @@
                 <source>Using hidden overlay characters is not allowed.</source>
                 <target>Using hidden overlay characters is not allowed.</target>
             </trans-unit>
+            <trans-unit id="110">
```

Comment:
```suggestion
            <trans-unit id="111">
```

As we already have 110 (you will get it if you rebase on current 7.1).

Reply:
Done, we are already at 112